news/_posts/2020-07-09-product-features-methodology-at-prestashop.md

```diff
@@ -80,8 +80,6 @@ Lastly, once you know what to focus on, and what you should work on next, it is
 
 All the product development process at PrestaShop is summarized in the following chart.
 
-![Product development steps at PrestaShop](/assets/images/2020/05/product-development-steps.png)
-
```

Comment:
Delete all lines please, there is no following chart in the end, thank you very much!

Reply:
From 81 to 84.